src/bacnet/bacdef.h

```diff
@@ -119,6 +125,8 @@
 /* largest BACnet Instance Number */
 /* Also used as a device instance number wildcard address */
 #define BACNET_MAX_INSTANCE (0x3FFFFF)
+// based on 169 alarms currently in App Charlie
+#define BACNET_MAX_SUPPORTED_INSTANCES 200
```

Comment:
Purely for my own curiosity, is this the maximum number of objects of each object type that we will be able to support?